### DISCUSSION TITLE
多语言发布日志询问

### DISCUSSION THREAD

**Discussion by lizhancheng** [ant-design/ant-design-mobile#5930] [General]
很好奇ant-design-mobile的多语言发布日志CHANGELOG，这里冒昧地问一下😅

**Comment by miracles1919** [comment]
https://mobile.ant.design/zh/guide/i18n 这里有支持的语言

单独的 changelog 是没有的，可以去 [releases](https://github.com/ant-design/ant-design-mobile/releases) 搜

**Reply by lizhancheng** [reply]
> https://mobile.ant.design/zh/guide/i18n 这里有支持的语言
> 
> 单独的 changelog 是没有的，可以去 [releases](https://github.com/ant-design/ant-design-mobile/releases) 搜

我是想问ant-design-mobile的CHANGELOG是怎么做多种语言的，比如现在的CHANGELOG有英文和中文，这种是怎么做的

**Reply by miracles1919** [reply]
mobile 没有做 CHANGELOG 吧

**Reply by lizhancheng** [reply]
> mobile 没有做 CHANGELOG 吧

 有吧，就是这个CHANGELOG，想知道这种双语言的这么做呢
<img width="511" alt="image" src="https://user-images.githubusercontent.com/8544620/217476217-29e0a83a-f1f0-4fd7-9b69-45752d329d9e.png">
